UnitySetup/UnitySetup.psm1

```diff
@@ -465,9 +465,9 @@ function Install-UnitySetupInstance {
                 if (!(Test-Path $destDirectory -PathType Container)) {
                     New-Item "$destDirectory" -ItemType Directory | Out-Null
                 }
-            }
 
-            (New-Object System.Net.WebClient).DownloadFile($downloadSource, $downloadDest)
+                (New-Object System.Net.WebClient).DownloadFile($downloadSource, $downloadDest)
```

Comment:
This should use `$downloadSource[$i]` and `$downloadDest[$i]`

Reply:
Just wanted to be clear. Tried this locally running @jwittner 's commands and was seeing those same errors. Changing them on my computer to what I stated above resolved those errors. :)

Reply:
ah!  copy paste error and didn't check.   thanks for calling that out.  had it the same way in my testing.